back/engines/commercial/idea_custom_fields/app/controllers/idea_custom_fields/web_api/v1/admin/idea_custom_fields_controller.rb

```diff
@@ -9,6 +9,8 @@ def initialize(errors)
     attr_reader :errors
   end
 
+  class UpdatingSurveyWithResponsesError < StandardError; end
```

Comment:
This comment applies to the whole PR, not only this file.

I would avoid using the the words "survey" and "responses". Isn't it better to use "form" and "inputs"?

Reply:
Yes, sounds good :)